### ISSUE TITLE
Sort spaces in invitees section 

### ISSUE THREAD

**Issue by morchickit** [hasadna/Open-Knesset#693]
Currently it's machine readable (lobbyist are stated correctly), but not human readable.

See example here - https://oknesset.org/committee/meeting/8208/?page=1

**Comment by OriHoch** [comment]
I think the problem is only with old meetings, newer meetings look fine - 
https://oknesset.org/committee/meeting/12402/

**Comment by morchickit** [comment]
Yes, this is really low priority, but I thought I should report on it anyway
